csharp/ql/src/semmle/code/csharp/PrintAst.qll

```diff
@@ -31,46 +31,27 @@ private predicate shouldPrint(Element e, File f) {
   exists(PrintAstConfiguration config | config.shouldPrint(e, f))
 }
 
-private predicate isInsideUnneededAttributable(Attributable attributable) {
-  isInsideUnneededType(attributable.(Field).getDeclaringType()) or
-  isInsideUnneededParameterizable(attributable.(Parameter).getDeclaringElement()) or
-  isInsideUnneededCallable(attributable.(Callable)) or
-  isInsideUnneededType(attributable.(DeclarationWithAccessors).getDeclaringType()) or
-  isInsideUnneededType(attributable.(ValueOrRefType))
+private predicate isImplicitExpression(ControlFlowElement element) {
+  element instanceof Expr and element.(Expr).isImplicit() and not exists(element.getAChild())
```

Comment:
`element instanceof Expr` is not needed as it is implied from the infix cast in `element.(Expr).isImplicit()`.